.gitignore

```diff
@@ -1,8 +1,176 @@
-*/node_modules/
-/development/.ipfs
+# Common #
+##########
+.env
+.idea
+.sass-cache
+build/
```

Comment:
Will we have any webpack files in the root ? wouldn't they be under origin-dapp/, origin-js/ etc... ?